tpdatasrc/tpgamefiles/rules/d20_combat/damage_critter.py

```diff
@@ -128,7 +134,7 @@ def deal_attack_damage(attacker, tgt, d20_data, flags, action_type):
     damage_critter(attacker, tgt, evt_obj_dam)
 
     #Play damage particle effects
-    evt_obj_dam.damage.play_pfx(tgt)
+    evt_obj_dam.damage_packet.play_pfx(tgt)
```

Comment:
??? Same here

Reply:
I have fixed this, you added the function play_pfx in tpdp and unstubbed it.

The problem is damage (which is correct in c++)  but needs to be damage_packet in python.

Reply:
To be clear, I tested this yesterday quite a bit, atm the deal_attack_damage function basically works as it should expect the stubbed parts (action type and a sound call). The damage looks fine.

Testing environment: I add the condition Unconscious to a party member, let him fall_down()  and then a second party member uses Coup de Grace.

What I haven't tested so far is the concealment and the deflect arrow part, which is on my ToDo for today.

Reply:
Hmm ok, I recall testing it but maybe I added these afterwards. Will check.

Reply:
Ty :)
If you find time, could you please also add a hook to the deal_spell_damage so I can start testing it? I think it looks pretty good, only stub I have (besides the action_type) would be the Metag Magic

Reply:
I have a proposal for you ;) can you copy the stuff I did in deal_attack_damage?

Reply:
Just to be sure, I need both correct?

https://github.com/GrognardsFromHell/TemplePlus/blob/master/TemplePlus/damage.cpp#L421
https://github.com/GrognardsFromHell/TemplePlus/blob/master/TemplePlus/damage.cpp#L537-L549

Reply:
Yeah